extensions/2.0/Vendor/ADOBE_materials_clearcoat_tint/schema/glTF.ADOBE_materials_clearcoat_tint.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "ADOBE_materials_clearcoat_tint glTF extension",
+    "type": "object",
+    "description": "glTF extension that defines the colour tint of the clearcoat.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "clearcoatTintFactor": {
+            "type": "array",
+            "items": {
+                "type": "number",
+                "minimum": 0.0,
+                "maximum": 1.0
+            },
+            "description": "The transmittance of the clearcoat layer.",
+            "default": [ 1.0, 1.0, 1.0 ],
+            "minItems": 3,
+            "maxItems": 3,
+            "gltf_detailedDescription": "The colour of light allowed to be transmitted through the clearcoat layer of the material. A value of black means no light passes through. A value of white means all light passes through."
```

Comment:
Like other glTF factors, this should be expressed as linear, even though the associated texture is sRGB.

Reply:
Indeed. Thank you!